integration/src/main/java/scripts/testscripts/DatasetOperations.java

```diff
@@ -161,12 +164,26 @@ private void previewUserJourney(StorageSystem storageSystem, String sourceId)
     datasetId = null;
   }
 
+  private ObjectNode expectedMetadataResponse(ObjectNode metadata, StorageSystem storageSystem) {
+    ObjectNode expectedMetadata = metadata.deepCopy().put("id", datasetId.toString());
+    if (storageSystem.equals(StorageSystem.TDR)) {
+      expectedMetadata
+          .put("phsId", "1234")
+          .put(
+              "dcat:accessURL",
+              String.format(
+                  "https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=%s",
```

Comment:
I don't think we should hardcode this path in the test, as the path contents isn't part of the API; the API specifies that the property must be present but doesn't specify what its contents are. Having the test rely on the contents means that the test must be updated if the contents ever change. There are a few ways to match json objects, we could check for each property separately or try using one of the json matching libraries, like https://github.com/skyscreamer/JSONassert.